### PR TITLE
[AN-4265] Remove obfuscation for all builds

### PR DESCRIPTION
**Pull request by xuanzhaopeng**
Remove obfuscation for all builds.
#### APK
[Download build #7585](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7585/artifact/build/artifact/wire-dev-PR154-7585.apk)
[Download build #7587](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7587/artifact/build/artifact/wire-dev-PR154-7587.apk)
[Download build #7589](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7589/artifact/build/artifact/wire-dev-PR154-7589.apk)
[Download build #7590](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7590/artifact/build/artifact/wire-dev-PR154-7590.apk)
[Download build #7611](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7611/artifact/build/artifact/wire-dev-PR154-7611.apk)
[Download build #7617](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7617/artifact/build/artifact/wire-dev-PR154-7617.apk)

### CODE REVIEW

**Comment by hakonbo** [comment]
build again

File: app/proguard-rules.txt [hunk 1]
```diff
@@ -16,11 +16,6 @@
 #   public *;
 #}
 
--keepclassmembers class com.waz.zclient.views.** {
```
Comment (hakonbo) [inline app/proguard-rules.txt]: Might as well keep this if we decide to do obfuscation again.

File: app/proguard-rules.txt [hunk 2]
```diff
@@ -29,7 +24,6 @@
 -keep class com.waz.call.CaptureDevice { *; }
 -keep class com.waz.media.manager.** { *; }
 -keep class com.waz.service.call.** { *; }
--keep class com.waz.zclient.controllers.loadtimelogger.LoadTimeLoggerController { *; }
```
Comment (hakonbo) [inline app/proguard-rules.txt]: Keep

File: app/proguard-rules.txt [hunk 3]
```diff
@@ -60,12 +54,6 @@
     public static final ** CREATOR;
 }
 
-# keep setters in controllers so that animations can still work
```
Comment (hakonbo) [inline app/proguard-rules.txt]: keep

**Comment by vova-lohika** [comment]
build again

**Pull request merged**